modules/project_services/README.md

```diff
@@ -0,0 +1,42 @@
+# project_services
+
+This optional module is used to enable project APIs in your project. By default,
+this module will enable the following APIs:
+- `compute.googleapis.com`
+- `iam.googleapis.com`
+
+The list of APIs to be enabled can be customized with the `activate_apis`
+variable.
+
+Google Project Factory already does this [here](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/main.tf#L112), but this module allows you to enable APIs on projects that were not created with Project Factory.
+
+## Prerequisites
+
+1. Service account used to run Terraform has permissions to manage project APIs: 
+[`roles/serviceusage.serviceUsageAdmin`](https://cloud.google.com/iam/docs/understanding-roles#service-usage-roles) or `Owner`
+
+## Usage
```

Comment:
People should not directly invoke the module. Instead include an example of using it inside a Terraform config.